### ISSUE TITLE
Ship miss_hit with built in Fish completions

### ISSUE THREAD

**Issue by EmilySeville7cfg** [florianschanda/miss_hit#249]
**What kind of feature is this?**

* New feature in MISS_HIT

**Your MATLAB/Octave environment**

* MATLAB or Octave
* Which release/version

**MISS_HIT component affected**

* Style checker
* Code metrics

**Describe the solution you'd like**

I suggest you to include [these Fish completions](https://github.com/fish-shell/fish-shell/pull/8499) to miss_hit. As an author you know better when and how to update them. I can help you in updating of course. :)

**Comment by florianschanda** [comment]
@EmilySeville7cfg I am not sure what action I have to do here. Can you explain it to me (I have no idea what fish is). Do I need to add some files somewhere? Do I need to do something with an external project?

**Comment by EmilySeville7cfg** [comment]
Yes, you have to include [these](https://github.com/fish-shell/fish-shell/pull/8499/files) Fish complections for miss_hit. Installer can place them into [`/usr/share/fish/vendor_completions.d`](https://fishshell.com/docs/current/completions.html#where-to-put-completions) directory. The helper script `__fish_list_python_encodings.fish` can be placed in [`/usr/share/fish/vendor_functions.d`](https://fishshell.com/docs/current/language.html?highlight=functions#autoloading-functions).

**Comment by florianschanda** [comment]
I have _no_ idea if I can put things in /usr with a PyPi package, but I guess I could also start rolling debian packages... Hmmm

**Comment by florianschanda** [comment]
I have renamed the ticket to that effect.

And yeah, I am 99% sure I cannot install stuff into /usr like that with PyPi.

**Comment by EmilyGraceSeville7cf** [comment]
I guess I can create a PR to put these completions here and write some instructions on how to install them properly.
